testsupport/space.go

```diff
@@ -219,7 +219,9 @@ func verifyResourcesProvisionedForSpace(t *testing.T, hostAwait *wait.HostAwaiti
 		ns, err := targetCluster.WaitForNamespace(t, space.Name, tier.Spec.Namespaces[0].TemplateRef, space.Spec.TierName, wait.UntilNamespaceIsActive())
 		require.NoError(t, err)
 		require.Contains(t, ns.Labels, toolchainv1alpha1.WorkspaceLabelKey)
+		require.Contains(t, ns.Labels, "appstudio.redhat.com/workspace_name")
 		assert.Equal(t, space.Name, ns.Labels[toolchainv1alpha1.WorkspaceLabelKey])
+		assert.Equal(t, space.Name, ns.Labels["appstudio.redhat.com/workspace_name"])
```

Comment:
I'm not sure if we really need this part here. We already verify the provisioned labels as part of the checks.go, which should be enough

Reply:
yes this is a repeat check. let me remove it.

Reply:
@MatousJobanek I'm wondering if we still need that if statement for the `appstudio` tier ? 
We could probably check the `toolchainv1alpha1.WorkspaceLabelKey` as part of the `func (a *appstudioTierChecks) GetNamespaceObjectChecks` and drop this specific case ?

Reply:
no, we don't need that - let's drop it as part of your cleanup story :+1: